plugins/modules/appliance_monitoring_info.py

```diff
@@ -82,834 +82,6 @@
 """
 
 RETURN = r"""
-# content generated by the update_return_section callback# task: Get the list of the monitored items
-value:
-  description: Get the list of the monitored items
-  returned: On success
-  sample:
-  - category: com.vmware.applmgmt.mon.cat.disk
-    description: com.vmware.applmgmt.mon.descr.disk.read.rate.dm-1
```

Comment:
Examples generation does not work at the moment in the content builder, so we will end up deleting examples. We should not delete/alter the examples that already exist unless they receive an update.